docs/application/native/guides/complications/watchface-complication-provider.md

```diff
@@ -49,37 +58,55 @@ The complication data can be set using the following APIs. These APIs can be use
 
 | API | Description |
 |--------|-----------------|
-| `watchface_complication_provider_data_set_title()` | Set the complication title. |
-| `watchface_complication_provider_data_set_short_text()` | Set the short text type data. |
-| `watchface_complication_provider_data_set_long_text()` | Set the long text type data. |
-| `watchface_complication_provider_data_set_image_path()` | Set the image path type data. |
-| `watchface_complication_provider_data_set_ranged_value()` | Set the ranged value type data. |
-| `watchface_complication_provider_data_set_icon_path()` | Set the icon path type data. |
-| `watchface_complication_provider_data_set_timestamp()` | Set the timestamp type data. |
-| `watchface_complication_provider_data_set_extra_data()` | Set the extra data. |
+| `watchface_complication_provider_data_set_title()` | Sets the complication title. |
+| `watchface_complication_provider_data_set_short_text()` | Sets the short text type data. |
+| `watchface_complication_provider_data_set_long_text()` | Sets the long text type data. |
+| `watchface_complication_provider_data_set_image_path()` | Sets the image path type data. |
+| `watchface_complication_provider_data_set_ranged_value()` | Sets the ranged value type data. |
+| `watchface_complication_provider_data_set_icon_path()` | Sets the icon path type data. |
+| `watchface_complication_provider_data_set_timestamp()` | Sets the timestamp type data. |
+| `watchface_complication_provider_data_set_extra_data()` | Sets the extra data. |
+| `watchface_complication_provider_timeinfo_create()` | Creates the time information handle. |
+| `watchface_complication_provider_timeinfo_set_timezone()` | Sets the timezone. For example, UTC+8 |
+| `watchface_complication_provider_timeinfo_set_timezone_id()` | Sets the timezone ID. For example, Asia/Seoul |
+| `watchface_complication_provider_timeinfo_set_timezone_country()` | Sets the timezone country. For example, Korea |
+| `watchface_complication_provider_timeinfo_set_timezone_city()` | Sets the timezone city. For example, Seoul |
+| `watchface_complication_provider_timeinfo_destroy()` | Destroys the time information handle. |
+| `watchface_complication_provider_data_set_timeinfo()` | Sets the time information data. |
+
+> **Note**
+>
+> `watchface_complication_provider_data_set_timestamp()` is deprecated since Tizen 5.5.
+> Instead, use `watchface_complication_provider_timeinfo_create()` and `watchface_complication_provider_timeinfo_set_timezone_id`.

```

Comment:
I think [this comment](https://github.com/Samsung/tizen-docs/pull/856#discussion_r354083874) is missing..

>**mijin85cho**
>How about adding () and a period to watchface_complication_provider_timeinfo_set_timezone_id?
watchface_complication_provider_timeinfo_set_timezone_id().

Would you please check it?

Also, after @gotoextreme updates this PR,
@rishu-samsung or @nawab-samsung, would you please review this PR again?

Reply:
> I think [this comment](https://github.com/Samsung/tizen-docs/pull/856#discussion_r354083874) is missing..
> 
> > **mijin85cho**
> > How about adding () and a period to watchface_complication_provider_timeinfo_set_timezone_id?
> > watchface_complication_provider_timeinfo_set_timezone_id().
> 
> Would you please check it?
> 
> Also, after @gotoextreme updates this PR,
> @rishu-samsung or @nawab-samsung, would you please review this PR again?

@mijin85cho Sure, will review this PR once it is updated by @gotoextreme